--- a/src/hash.c
+++ b/src/hash.c
@@ -58,1 +58,1 @@
-#define HASH_FUNC(_key) { register int z = IS5, x = _key.i ; \
+#define HASH_FUNC(_key) { int z = IS5, x = _key.i ; \
@@ -63,1 +63,1 @@
-#define DELTA(_key)   { register int z = IS7, x = _key.i ; \
+#define DELTA(_key)   { int z = IS7, x = _key.i ; \
--- a/src/ksort.h
+++ b/src/ksort.h
@@ -41,1 +41,1 @@
-#define KSORT_SWAP(type_t, a, b) { register type_t t=(a); (a)=(b); (b)=t; }
+#define KSORT_SWAP(type_t, a, b) { type_t t=(a); (a)=(b); (b)=t; }
--- a/hifiasm/ksort.h
+++ b/hifiasm/ksort.h
@@ -41,1 +41,1 @@
-#define KSORT_SWAP(type_t, a, b) { register type_t t=(a); (a)=(b); (b)=t; }
+#define KSORT_SWAP(type_t, a, b) { type_t t=(a); (a)=(b); (b)=t; }
